components/button/Button.js

```diff
@@ -83,13 +83,14 @@ class Button extends PureComponent {
         </span>
       ),
       icon && iconPlacement === 'right' && icon,
-      processing && (
-        <LoadingSpinner
-          className={theme['spinner']}
-          color={this.getSpinnerColor()}
-          size={size === 'small' ? 'small' : 'medium'}
-        />
-      ),
+      processing &&
+        level !== 'link' && (
```

Comment:
Any specific reason why a LinkButton should not be able to have a processing state?

Reply:
Comment to his commit is
> as we do not have an appropriate spinner style for it yet
I'm not sure if that's an issue though? I'd say just allow it to do the style we have now? Or is Stephen radically against this?

Reply:
At this moment the `LoadingSpinner` comes in only two colours,
`"white"` and `"teal"`
![screen shot 2018-09-25 at 17 26 52](https://user-images.githubusercontent.com/23736202/46025276-de572980-c0e8-11e8-9131-ce543b229910.png)
![screen shot 2018-09-25 at 17 26 52](https://user-images.githubusercontent.com/23736202/46025272-dbf4cf80-c0e8-11e8-807b-f839c2e5b633.png)

Normally the spinner takes the same color as the text does, for the `"link"` level this would be `aqua`, adding this color to the `LoadingSpinner` felt something for another PR to me. (the inverse `"link"` can actually take the already available white spinner).

Also the original `"LinkButton"` does not have a processing state (but I also  think it should have one).

Reply:
Go for `a` or `c`, and I would even prefer `a`.

Reply:
`a` it is.